src/main/java/seedu/address/model/person/Rating.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an internship's Rating in the tracker.
+ * Guarantees: immutable; is valid as declared in {@link #isValidRating(String)}
+ */
+public class Rating {
```

Comment:
Perhaps, we may consider using Enum for this, but in terms of usability, I do agree with having a more flexible rating